src/quo2/components/info/lowest_price.cljs

```diff
@@ -0,0 +1,72 @@
+(ns quo2.components.info.lowest-price
+  (:require [quo.react-native :as rn]
+            [clojure.string :as string]))
+
+(def centrify {:style {:flex-direction  :row
+                       :justify-content :center
+                       :align-items     :center}})
+
+(defn border-alignment [value-bg-color]
+  {:style {:align-self         :flex-start
+           :padding-horizontal 16
+           :padding-vertical   2
+           :flex-grow          0.25
+           :border-radius      4
+           :background-color   value-bg-color}})
+
+(def line-dots-props
+  {:ellipsize-mode  :middle
+   :number-of-lines 1
+   :font-size       10
+   :line-height     2})
+
+(defn dots
+  "Returns the dots"
+  [few-o-dots?]
+  (let [dots-text (->> ". "
+                       (repeat (if few-o-dots?
+                                 16
+                                 48))
+                       string/join
+                       (str " "))]
+    dots-text))
+
+(defn dots-comp
+  "Returns the dots adding their styles"
+  [lots-o-dots?]
```

Comment:
here lots, but dots expects few

Reply:
Correct, I messed up that logic and fixed it in the function above .. Thanks for noticing it.

Reply:
Done